cnesreport.properties

```diff
@@ -1,12 +1,18 @@
 category=Visualization/Reporting
 description=CNES plugin that allows users to download a bundle of project reports in multiple formats.
 homepageUrl=https://github.com/cnescatlab/sonar-cnes-report
-archivedVersions=4.0.0,4.1.0
-publicVersions=4.1.1
+archivedVersions=4.0.0,4.1.0,4.1.1
+publicVersions=4.1.2
 
 defaults.mavenGroupId=fr.cnes.sonarqube.plugins
 defaults.mavenArtifactId=cnesreport
 
+4.1.2.description=Fix issues on the project name on Windows, a warning in logs, Security Hotspots categories and missing information for external issues.
+4.1.2date=2022-06-13
```

Comment:
```suggestion
4.1.2.date=2022-06-13
```

Reply:
Oh crap... a silly mistake. Thanks for noticing it ! :)